python/mxnet/gluon/block.py

```diff
@@ -1248,7 +1248,11 @@ def export(self, path, epoch=0, remove_amp_cast=True):
             will be created, where xxxx is the 4 digits epoch number.
         epoch : int
             Epoch number of saved model.
-
+        remove_amp_cast : bool, optional
+            Whether to remove the amp_cast and amp_multicast operators, before exporting the model.
+        allow_extra : bool, optional
```

Comment:
What is the use-case to save parameters that are not used by the symbol? Do we need this option?

Reply:
I think it's not necessary.